test/article.test.js

```diff
@@ -175,17 +175,17 @@ describe('/articles Route', () => {
                 .get('/articles/new')
                 .query({ lastDate: longAgoDate })
                 .end((err, res) => {
-                    res.body.should.be.array
+                    res.body.should.be.an('array')
                     res.body.should.be.empty
                     done()
                 })
         })
     })
 })
 
-function checkReponse(res) {
+function checkResponse(res) {
     res.should.have.status(200)
-    res.body.should.be.arrary
+    res.body.should.be.an('array')
```

Comment:
So I noticed that array was spelt wrong here but the test still passed. Further research showed me that we should be asserting the type using .an('type). 

In my tests a object passed as an array using the should.be.array method but this way of doing things fixed everything.